<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consumer</class>
 <widget class="QWidget" name="Consumer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>1085</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableFlights">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>721</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSearch">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>80</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBook">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonViewBookings">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>180</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ViewBookings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>230</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="broadcast">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
